ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/generators/responseparser/AllocateJobChunkParserGenerator.java

```diff
@@ -35,7 +35,8 @@
     @Override
     public ImmutableList<ResponseCode> toResponseCodeList(
```

Comment:
There are several functions here where the parameter was added. I don't think AllocateJobChunkParserGenerator.toResponseCodeList is called often enough for this to clean up the code.  In general, this class inherits from BaseResponseParserGenerator, which does need the added parameter.